src/txmempool.cpp

```diff
@@ -831,11 +832,17 @@ TxMempoolInfo CTxMemPool::info(const GenTxid& gtxid) const
 
 TxMempoolInfo CTxMemPool::info(const uint256& txid) const { return info(GenTxid{false, txid}); }
 
-void CTxMemPool::PrioritiseTransaction(const uint256& hash, const CAmount& nFeeDelta)
+bool CTxMemPool::PrioritiseTransaction(const uint256& hash, const CAmount& nFeeDelta)
 {
+    if (nFeeDelta == std::numeric_limits<CAmount>::min()) {
+        return false;
+    }
     {
         LOCK(cs);
         CAmount &delta = mapDeltas[hash];
+        if (AdditionOverflow(delta, nFeeDelta)) {
```

Comment:
This check isn't complete. You'd have to check all ancestors and descandants as well. Also, I don't understand why this can't be `FeeDeltaRange` https://github.com/bitcoin/bitcoin/pull/20089/files#r557249710

Reply:
Implemented an alternative that also checks all ancestors and descendants here: #23418